enricher/mappings/mappings.go

```diff
@@ -0,0 +1,11 @@
+package mappings
+
+const (
+	RESOURCE_CLOUD_ACCOUNT_ID = "cloud.account.id"
+	RESOURCE_ACCOUNT_GROUP    = "account_group"
```

Comment:
👋 @wimnat 

How should we label the account group? Would this be fine under `resource`

Reply:
I think it works under `resource` alongside the account ID. 👍

Reply:
We decided that the account group function will fall under as `canva.account.function`